hooks/queries/nft.ts

```diff
@@ -57,3 +61,40 @@ export const fetchNFTbyMint = (connection: Connection, pubkey: PublicKey) => {
     },
   })
 }
+
+export const useOwnerVerifiedCollectionsQuery = (owner: PublicKey) => {
+  const { connection } = useConnection()
+  const { data: ownedNfts } = useDigitalAssetsByOwner(owner)
+  const network = getNetworkFromEndpoint(connection.rpcEndpoint)
+  const verifiedNfts = filterAndMapVerifiedCollections(ownedNfts)
+
+  const enabled = owner !== undefined && ownedNfts !== undefined
+  const query = useQuery({
+    queryKey: enabled ? nftQueryKeys.byMint(network, owner) : undefined,
+    queryFn: async () => {
+      if (!enabled) throw new Error()
+
+      const verifiedCollections = {}
+      for (const collectionKey in verifiedNfts) {
```

Comment:
it would be better to do this in parallel using Promise.all

Reply:
Changed to Promise.all